handlers/callback.ts

```diff
@@ -260,6 +260,9 @@ cq.callbackQuery("remove_repeated_spaces", async (ctx) => {
   while (/ {2,}/.test(newText)) {
     newText = newText.replace(/ {2,}/g, " ");
   }
+  while(/^ /.test(newText)) {
+    newText = newText.replace(/^ /, "");
+  }
```

Comment:
This poses unexpected behavior. Spaces before list items are intentional.

Reply:
fuck :/